src/workers/notifications/userStreakResetNotification.ts

```diff
@@ -48,12 +49,14 @@ const worker: TypedNotificationWorker<'api.v1.user-streak-updated'> = {
       return;
     }
 
-    const ctx: NotificationStreakContext = {
+    const restorationValidUntil = redisExpiryTime * 1000; // unix timestamp in ms
```

Comment:
`redisExpiryTime` can be null or empty I suppose?

Reply:
Can we handle it somehow and add test case on what happens when it is

Reply:
let's just add a check if its not in the future we early return as well

Reply:
Cool. I'll add it in the conditions.